ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -85,6 +102,235 @@ const triggerSet: TriggerSet<Data> = {
+
+        // Polys 2, 3, 5, and 6 involve tethers. We only care about which
+        // tiles have tethers, not which tiles are tethered together.
+        // This is because there are only two types of tiles, and swaps
+        // are only done between tiles of opposite types.
+        const tetheredTiles = [];
+        if (data.tileTethers.length >= 1) {
+          for (const tile of data.combatantData) {
+            // x, y = 85, 95, 105, 115 (with a little variance)
+            // map to ([0, 1, 2, 3] and [0, 10, 20, 30]
+            const x = Math.floor((tile.PosX - 83) / 10); // add in a -2/+8 buffer in case of goofy pos data
+            const y = Math.floor((tile.PosY - 83) / 10) * 10; // add in a -2/+8 buffer in case of goofy pos data
+            const idx = x + y;
+            if (unsafeMap[idx] === undefined)
+              return;
+            tetheredTiles.push(idx);
+          }
+        }
+
+        // modifiers used to calculate unsafeMap indexes to be removed for each type of tile
+        const relCrossTiles = new Int8Array([-30, -20, -10, -3, -2, -1, 1, 2, 3, 10, 20, 30]);
+        const relDiagonalTiles = new Int8Array([-33, -27, -22, -18, -11, -9, 9, 11, 18, 22, 27, 33]);
+
+        for (const effect of data.mapEffects) {
+          if (mapLookup[effect.location] === undefined)
+            return;
+
+          const startTile: number = mapLookup[effect.location]!;
+          const isTethered: boolean = tetheredTiles.includes(startTile);
+
+          if (unsafeMap[startTile] !== undefined)
+            delete safe[unsafeMap[startTile]![1]]; // delete tile where effect appears, as it will always be unsafe
+          if ((effect.flags === crossTileFlags && !isTethered) || (effect.flags === diagonalTileFlags && isTethered)) {
+            relCrossTiles.forEach((tileMod) => {
+              const deleteTile: number = startTile + tileMod;
+              if (unsafeMap[deleteTile] !== undefined)
+                delete safe[unsafeMap[deleteTile]![1]];
+            });
+          } else if ((effect.flags === diagonalTileFlags && !isTethered) || (effect.flags === crossTileFlags && isTethered)) {
+            relDiagonalTiles.forEach((tileMod) => {
+              const deleteTile: number = startTile + tileMod;
+              if (unsafeMap[deleteTile] !== undefined)
+                delete safe[unsafeMap[deleteTile]![1]];
+            });
+          } else {
+            return;
+          }
+        }
+
+        const safeTiles: string[] = Object.keys(safe);
+        const [safe0, safe1] = safeTiles;
+
+        if (safe0 === undefined)
+          return;
+
+        const poly6Map: { [l: string]: string } = {
+          insideNW: 'insideSE',
+          insideNE: 'insideSW',
+          insideSE: 'insideNW',
+          insideSW: 'insideNE',
+        };
+
+        switch (data.polyInstance) {
+          case 1: // four safe spots: two inside, two outside.  call only the inside ones.
```

Comment:
Interesting - the way my various groups do it and the way I've seen the few times I've done it in PF is that one healer group takes the north inside tile, and the other takes the south.  But if you're seeing it resolve mostly east/west splits, then I think that's what we should call (and either way, calling inside west/outside east gives more fulsome info).  And yes, completely agree with combining it with exocleaver.

Reply:
> Do you think it'd be worth combining this with `P6S Exocleaver Healer Groups` too, e.g. "Healer groups (inside west/outside east)" or however we'd word it.

Actually, testing this tonight, I think the combined call "Healer Groups (Inside North/Outside South)" is a bit too long/wide for a single output on screen.  I think it might make sense to either have the inside north/outside south still called by the poly trigger, or to have the Exocleaver call do a response: and put the healer groups up as alertText and the directions up as infoText.  Thoughts?

Reply:
I am personally not sure what people do.  I'm in a static, but I also looked at some toolboxes and everybody seemed to split E/W.  I think that's fine to keep it split up.  I think even if both stacks are on the inside side, saying "inside E/outside W" will still be clear where to go.

I guess my other feedback is that if it's not going to be combined, then I think "safe tiles: X" is a bit verbose, probably especially for folks who are using TTS.  What do you think about dropping the "safe tiles" part, or do you think it's confusing without it?

Reply:
> What do you think about dropping the "safe tiles" part, or do you think it's confusing without it?

Good point - I use the text alerts not tts, so wasn't thinking about that.  I think we can drop "safe tiles" and people will be able to get quickly what it refers to, especially as it fires during the mapeffects and before other mechanics start happening.